Fix lesson padding on bottom

VincentCATILLON opened the pull request:
## Detailed purpose of the PR

<img width="330" alt="Capture d’écran 2019-03-22 à 15 32 40" src="https://user-images.githubusercontent.com/8419208/54830954-c50af080-4cb9-11e9-9055-855a0026f5c3.png">

### Result and observation

<img width="329" alt="Capture d’écran 2019-03-22 à 15 48 20" src="https://user-images.githubusercontent.com/8419208/54831058-f2f03500-4cb9-11e9-9e7f-34531ee753db.png">

- [ ] **Breaking changes ?**
- [ ] **Extra dependency ?**

### Testing Strategy

- [x] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

adamska27 commented:
c'est normal qu'en changeant "lesson" tu as un snapshot "correction" qui soit update?

VincentCATILLON commented:
@adamska27 Yes, introduced by disabling `Keep branch up to date` in repository settings.

Introduced there: https://github.com/CoorpAcademy/mobile/pull/194
